web/src/poll_modal.ts

```diff
@@ -56,11 +57,11 @@ export function poll_options_setup() {
     }, 0);
 }
 
-export function frame_poll_message_content() {
-    const question = $("#poll-question-input").val().trim();
+export function frame_poll_message_content(): string {
+    const question = $("#poll-question-input").val()!.toString().trim();
     const options = $(".poll-option-input")
         .map(function () {
-            return $(this).val().trim();
+            return $(this).val()?.toString().trim();
```

Comment:
No, use `$<HTMLInputElement>("input.poll-option-input")` and not `.toString()`.

Reply:
what about line 12? Is it correct?

Reply:
Nope..I think my idea was wrong.

Reply:
I am talking about line 12, the line that Tim mentioned.